src/compiler/reviews.md

```diff
@@ -1,4 +1,159 @@
-# Review policies
+Review Policy
+=============
+
+It is the purpose of code reviews to
+
+- **reduce** the risk of introducing **bugs** and usability and performance **regressions**,
+- keep our code **maintainable**, **readable**, and **documented**, and
+- ensure that changes are made with the big picture and **appropriate context in mind**.
+
+Reviewing accomplishes this by bringing in another set of eyes
+to look at a proposed change from a different perspective,
+which increases the chance of catching mistakes early and
+uncovering potential blind spots in the reasoning behind the change.
+
+
+## Basic Reviewing Requirements
+
+There are a number of requirements that need to be met in order for reviewing to be effective:
+
+- Reviewers must have a sufficient **understanding of the code** under review.
+    > This is important to help spot non-obvious, unintentional side effects of a given change.
+- Pull requests must provide (1) a concise high-level **description of the change** and (2) the **rationale** behind it.
+  For the rationale to be even more useful, authors are encouraged to list potential **points of contention**.
+    > Reviewing code is difficult and reviewers only have a limited amount of time to do it.
+    > Jump-starting the review process by not making the reviewer puzzle together the intention and context
+    > of a pull request will not only speed things up but also improve the quality of the review.
+- Reviewers must have a good idea on whether they are the **right person to approve** the change.
+    > Knowledge of the code under review is an obvious but not the only criterium for answering this question.
+    > The reviewer also needs to decide if they can make the decision alone or if the PR needs to go through
+    > the [major change process][mcp], if they can perform the review in a timely fashion,
```

Comment:
nit: not only timely, but also sufficiently thorough: a quick but not sufficiently thorough review is not what we want either.

Reply:
Done